6.4.remember_remember.py

```diff
@@ -0,0 +1,29 @@
+from PIL import Image
+import os.path
+
+
+def decipher(img):
```

Comment:
Decipher is a not a good function name - pick a better one for this usecase

Reply:
Done